.gitignore

```diff
@@ -0,0 +1,104 @@
+# Logs
+logs
```

Comment:
add .idea

Reply:
add tests to see that things are being cached, and that headers are returned in response